python_benchmarks/benchmark_layernorm.py

```diff
@@ -0,0 +1,148 @@
+from nvfuser import FusionDefinition, DataType
+from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
+from .core import runBenchmark
+import torch
+from .global_params import pytestmark, RTOL, ATOL
+
+
+def layernorm_fwd_fusion(
+    fd: FusionDefinition,
+    dtype: DataType,
+    eps: float = 1e-5,
+) -> None:
+    T0 = fd.define_tensor(
```

Comment:
In the cpp benchmark, we also have `weights` and `bias` as input, is there a reason we are using a different version? Same for backward.

Reply:
The `thunder` definition is probably missing that since I didn't explicitly initialize them.